src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -187,14 +187,14 @@ public static boolean isExpired(Date expiration) {
    * @param clientSecret the application client secret
    * @param redirectUri the registered redirect URI for the application
    * @param scope the permission scope requested
-   * @param development whether or not to operate in development mode
+   * @param testMode whether or not mode should be set to 'test'
```

Comment:
See comment in https://github.com/smartcar/java-sdk/pull/35#discussion_r216861953